access_group.go

```diff
@@ -0,0 +1,214 @@
+package cloudflare
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// AccessGroup defines a group for allowing or disallowing access to
+// one or more Access applications.
+type AccessGroup struct {
+	ID        string     `json:"id,omitempty"`
+	CreatedAt *time.Time `json:"created_at"`
+	UpdatedAt *time.Time `json:"updated_at"`
+	Name      string     `json:"name"`
+
+	// The include group works like an OR logical operator. The user must
+	// satisfy one of the rules.
+	Include []interface{} `json:"include"`
```

Comment:
This is now used in a couple of places; we might consider consolidating this struct at some point.